install/oddjob/com.redhat.idm.trust-fetch-domains.in

```diff
@@ -234,97 +193,95 @@ trusted_domain = trusted_domain_entry.single_value.get("cn").lower()
 # At this point if we didn't find trusted forest name, an exception will be raised
 # and script will quit. This is actually intended.
 
+rc = 0
+
 # Generate MIT Kerberos configuration file that potentially overlays
 # the KDC to connect to for a trusted domain to allow --server option
 # to take precedence.
-cfg_file, cfg = generate_krb5_config(trusted_domain, options.server)
+with ipautil.private_krb5_config(trusted_domain, options.server) as cfg_file:
+    if not (options.admin and options.password):
+        oneway_keytab_name = os.path.join("/var/lib/sss/keytabs/",
```

Comment:
I guess this is ok but we may want to consider adding this as a constant to ipaplatform. It only occurs in three places in the tree (one a comment).

Reply:
I am hesitant to turn this into a constant -- this is referencing SSSD internals that aren't going to change or aren't changeable. The constant would need to be somewhere in ipaplatform to be reused between tests and the actual code and that's exactly wrong place to put it because it is not modifiable for a platform.

Reply:
I decided to change this as suggested. ;)